src/cnd/axios_rfc7807_middleware.ts

```diff
@@ -15,6 +16,7 @@ export class Problem extends Error {
     this.detail = detail;
     this.instance = instance;
     this.title = title;
+    this.isProblem = true;
```

Comment:
:mega: This seems to be better expressed at the type-level, which it is because it is a `Problem` type.